Fix Percentiles Line chart with float values

dej611 opened the pull request:
## Summary

This PR addresses an issue with the ID parser for the line chart when percentiles have float values.

* :bug: Fix for float values series id format
* ✅ Add tests for the float values scenario

Fixes #47000 

### Checklist


- [x] [Unit or functional tests](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility) were updated or added to match the most common scenarios

brianseeders committed:
[CI] Add baseline trigger job

spalger committed:
rename visual_baseline -> baseline_capture

dej611 committed:
:white_check_mark: Add tests for line point series with float values

dej611 committed:
:bug: Fix float value id handling by point series

patrykkopycinski committed:
[Security] Add Timeline improvements (#71506)

cauemarcondes committed:
[Observability] Change appLink passing the date range (#71259)

* changing apm appLink

* changing apm appLink

* removing title from api

* adding absolute and relative times

* addressing pr comments

* addressing pr comments

* addressing pr comments

* fixing TS issues

* addressing pr comments

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

cauemarcondes committed:
[APM] Use status_code field to calculate error rate (#71109)

* calculating error rate based on status code

* fixing unit test

* addressing pr comments

* adding erroneous transactions rate

* adding erroneous transactions rate

* adding error rate to detail page

* fixing i18n

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/61277/)
* Commit: 17d7e6e69aaadf46c33cefcb8c1c8f915c0e6f9b

### Build metrics
> ✅ unchanged

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/61277/","number":"61277","commit":"17d7e6e69aaadf46c33cefcb8c1c8f915c0e6f9b"}]}
PIPELINE-->

pheyos committed:
[ML] Functional tests - disable DFA creation and cloning tests

elasticmachine commented:
Pinging @elastic/kibana-app (Team:KibanaApp)

Zacqary committed:
[Metrics UI] Remove UUID from Alert Instance IDs (#71335)

* [Metrics UI] Use alertId instead of uuid for alertInstanceIds

Zacqary committed:
[Logs UI] Remove UUID from Alert Instances (#71340)

* [Logs UI] Remove UUID from Alert Instances

* Fix bad template string

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

mistic committed:
[test] Skips test preventing promotion of ES snapshot #71612

Commit:
[Logs UI] Refine log entry row context button (#71260)

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

walterra committed:
[ML] Kibana API endpoint for histogram chart data (#70976)

- Introduces dedicated Kibana API endpoints as part of ML and transform plugin API endpoints and moves the logic to query and transform the required data from client to server.
- Adds support for sampling to retrieve the data for the field histograms. For now this is not configurable by the end user and is hard coded to 5000. This is to have a first iteration of this functionality in for 7.9 and protect users when querying large clusters. The button to enable the histogram charts now includes a tooltip that mentions the sampler.

loixlab committed:
[Index template wizard] Remove shadow and use border for components panels (#71606)

Commit:
[Security Solution] Hide timeline footer when Resolver is open (#71516)

* Hide the Timeline footer, in the event viewer, if Resolver is showing

pgayvallet committed:
use inclusive language (#71607)

lizozom committed:
[Search] Server side search API (#70446)

* [search] Refactor the way search strategies are registered/retrieved on the server

* Fix types and tests and update docs

* Fix failing test

* Move strategy name into options

* Remove FE strategies

* TypeScript of hell
delete search explorer

* Fix search interceptor OSS tests

* test cleanup

* fix

* return search wrapper

* initial api

* Shiny happy cleanup

* docs

* fix jest test

* simplify strategy registration

* fix rebase

* fix rebase

* fix backport

* types

* TS for strategy

* docs

Co-authored-by: Lukas Olson <olson.lukas@gmail.com>
Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

dej611 committed:
[Visualize] Fix export table for table export links (#71249)

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

smith committed:
[APM] Additional data telemetry changes (#71112)

* Add a date range of `now-1d` to the cloud query
* Add a timeout of 5m to all queries (we'll investigate using async queries to improve this in the future.)
* Factor out the date range filter into a variable
* Fix a bug with the `indices_stats` tasks when it doesn't return data
* Update the merge mapping script to create a migration file

jonathan-buttner committed:
[SecuritySolution][Resolver] Adding siem index and guarding process ancestry (#71570)

* Adding siem index and guarding process ancestry

* Fixing type errors

Commit:
Only fetch saved elements once (#71310)

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

Bamieh committed:
Update  getUsageForCollection (#71609)

legrego committed:
Require granted API Keys to have a name (#71623)

lcawl committed:
Fixes typo in siem_cloudtrail job description (#71569)

darnautov committed:
[ML] Anomaly swim lane embeddable navigation and filter actions (#71082)

* [ML] dragSelect as part of ExplorerSwimlane component

* [ML] use wrapper ref

* [ML] rename callback

* [ML] WIP open in anomaly explorer

* [ML] MlUrlGenerator unit tests

* [ML] WIP actions

* [ML] restore pagination

* [ML] fix fromPage on initial load

* [ML] fix cell selection, filter and time range actions

* [ML] update url generator params

* [ML] prevent label text selection on drag select

* [ML] fix types and unit tests

* [ML] fix embeddable init

* [ML] fix swim lane unit tests

* [ML] change action label, use filter action only for single cell click

* [ML] fix time range bounds

* [ML] fix TS issues

* [ML] fix pagination persistence

* [ML] use viewByFrom the embeddable input

lukeelmers committed:
[data.search.aggs] Remove `use_field_mapping` from top hits agg (#71168)

Kerry350 committed:
[Logs UI] Anomalies page dataset filtering (#71110)

Adds dataset filtering to logs anomalies page

alisonelizabeth committed:
[IM] Address data stream copy feedback (#71615)

cauemarcondes committed:
[Observability] landing page always being displayed (#71494)

* hides overview item when no other obs plugin is available.

* reverting tests changes

* reverting tests changes

* adding overview to observability tests

pmuellr committed:
[eventLog] prevent log writing when initialization fails (#71339)

resolves https://github.com/elastic/kibana/issues/68309

Previously, if the initialization of the elasticsearch resources failed
during initialization, the event logger would still try to write events.
Which is somewhat of a catastrophic failure, as typically the logger would
try writing to the alias name, but no alias exists, so a new index would
be created with the name of the alias.  Making it impossible to initialize
successfully later until that index was deleted.

The core initialization calls already returned success indicators, so this
PR just responds to those and prevents the logger from writing to the index
if initialization failed.

jgowdyelastic committed:
[ML] Changing all calls to ML endpoints to use internal user (#70487)

* [ML] Changing all calls to ML endpoints to use internal user

* updating alerting

* updating documentation

* [ML] Changing all calls to ML endpoints to use internal user

* updating alerting

* updating documentation

* fixing missed types

* adding authorization headers to endpoint calls

* correcting has privileges call

* updating security tests

* odd eslint error

* adding auth header to module setup

* fixing missing auth argument

* fixing delete DFA job permission checks

* removing debug test tag

* removing additional ml privilege checks

* adding authorization header to _evaluate

* updating alerting cluster client name

* code clean up

* changing authorizationHeader name

* updating alterting documentation

* fixing secondary credentials

* adding management links

* updating SIEM telemetry

* fixing merge conflicts

* granting access to index patterns

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

elizabetdev committed:
[Maps] 7.9 design improvements (#71563)

* Design improvements

* more improvements

* Updated snapshots

* Removing unecessary flex item

* Updated snapshot

* Addressing pr review

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

nchaulet committed:
[Ingest Manager] Enable ingest manager plugin by default. (#70955)

DianaDerevyankina committed:
Convert vega tests to jest (#71073)

* Convert vega tests to jest

Part of #57813

* Remove unused config

* Move assets to __test__ folder and remove unnecessary code

* clenup

* cleanup

* Revert default.spec.hjson file and mock default_spec

* Refactor some code

Co-authored-by: Alexey Antonov <alexwizp@gmail.com>

nickpeihl committed:
Update ems-landing-page-url to 7.9 (#71532)

paul-tavares committed:
Ignore equality check against the `manifest_version` of the full agent config (#71637)

VladLasitsa committed:
Migrate timelion to the NP. (#69160)

* Migrate timelion to the NP.

* fixed ci

* Fixed paths

* fixed UI settings

* Fixed ci

* fix CI

* Fixed some comments

* Fixed browser tests

* fixed state

* Fixed comments

* Fixed save expression

* Fixed navigation

* fix CI

* Fixed some problem

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>
Co-authored-by: Alexey Antonov <alexwizp@gmail.com>

nreese committed:
[Maps] expose registerLayerWizard and registerSource in maps plugin start (#71553)

nreese committed:
[Maps] get isClustered from count request instead of source data request (#71528)

* [Maps] get isClustered from count request instead of source data request

* better naming

* tslint

* review feedback

Commit:
Fix coordinate maps layers dropdown (#70609)

flash1293 committed:
[Graph] Ensure inclusive language (#71416)

cauemarcondes committed:
[APM] Respect default time range defined in Kibana Advanced Settings (#71464)

* using kibana settings on date picker

* fixing unit tests and use date picker component to update the query params

* fixing translations

angorayc committed:
[SIEM] Add scripts for on boarding prepackage timeline (#67496)

* add prepackaged timelines

* generate ndjson

* expose end api points

* fix types

* fix types

* fix unit test

* install prepackage timelines

* plumbing for prepackaged timelines

* read ndjson by line

* fix unit test

* update templates

* fix types

* fix types

* fix integration test

* update script

* name it back

* add timeline status into rule status api

* fix update messages

* fix unit tests

* fix integration test

* rename types

* update prepackaged timelines

* update prepackaged timelines script

* update scripts

* fix update for elastic template

* move timeline utils

* export timelines scripts

* update module path

* fix intefration test

* add delete all timelines script

* readme

* add get_timeline_by_templatetimeline_id

* add getTimelineByIdRoute

* rename file

* add unit test

* fix types

* fix types

* update readme

* fix error id

* fix unit test

* update path

* update i18n

* update readme

* load prepacked timelines by default

* add unit tests

* Update x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_timelines/README.md

Co-authored-by: Xavier Mouligneau <189600+XavierM@users.noreply.github.com>

* review

* update prepacked timelines

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>
Co-authored-by: Xavier Mouligneau <189600+XavierM@users.noreply.github.com>

MadameSheema committed:
updates exception empty prompt text (#71654)

mattkime committed:
Management API - simpler interface, remove app context usage (#71144)

Management API - simpler interface, remove app context usage, consolidate rendeing

patrykkopycinski committed:
[Security][Detections] Create Threshold-based Rule type (#71371)

Commit:
[Resolver] Remove Client side API limits (#71660)

cauemarcondes committed:
[Observability] Landing page: Add promo panel for Ingest Manager (#71213)

* adding ingest manager panel

* adding ingest manager panel

* changing url

* removing git conflicts

skh committed:
[Ingest Manager] Send kibana version on requests to package registry (#71443)

* Send kibana version on requests to registry search.

* Only use semver part of kibanaVersion.

* Adjust test.

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

yctercero committed:
[Security Solution][Lists] - Update exception comments logic in API (#71602)

### Summary

Updated the logic so that newly added exception item comments are shown as expected.

qn895 committed:
[ML] Anomaly Detection: Annotations enhancements (#70198)

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

rylnd committed:
Fix bug where lists "needs configuration" while index is being created (#71653)

The behavior here was that you'd be redirected to detections from
wherever you were, with no warning/indication.

When we knew we needed an index, and that we could create one,
needsConfiguration was incorrectly 'true' during the time between
realizing this fact and creating the index. That intermediate state is
now captured in needsIndexConfiguration, which is true if we either
can't create the index or we failed our attempt to do so.

shahzad31 committed:
[Uptime] Duration Anomaly Alert (#71208)

MadameSheema committed:
updates edit exception text save button (#71684)

gmmorris committed:
merged lodash imports (#71672)

This is just a code cleanup.
A previous PR accidentally added a second import of the same module into alerts_client.ts.
This PR corrects that.

jonathan-buttner committed:
[EPM][IngestManager][SecuritySolution] Correctly handle nested types (#71680)

* Correctly handling nested types

* Correct test names

joshdover committed:
[DOCS] Clarify 'fields' option in SO.find docs (#71491)

peteharverson committed:
[ML] Fix error toasts shown when starting or editing jobs (#71618)

* [ML] Fix error toasts shown when starting or editing jobs

* [ML] Adds toast_notification_service.ts file

* [ML] Fix Jest and type_check tests

* [ML] Alter check for statusCode in error object handling

* [ML] Fix errors Jest test

spalger committed:
skip flaky suite (#71713)

rylnd committed:
[Security Solution][Detections] Update telemetry to use ML contract (#71665)

* Update security solution telemetry to use ML providers

This interface recently changed and we're now able to use the ML
contract to retrieve these values. A few unnecessary arguments are
stubbed as we're in a non-user, non-request context.

* Simplify our capabilities stub assignment

This is more legible but still gets the point across; the intermediate
variable was explicit but ultimately unnnecessary.

* Update tests following telemetry refactor

We're not calling different methods, so our mocks need to change
slightly.

MadameSheema committed:
[SIEM][Timeline] Updates all events text timeline (#71701)

* updates 'All events' timeline text to 'All'

* updates jest test

* fixes test issue

neptunian committed:
[Ingest Manager] Refactor Package Installation (#71521)

* refactor installation to add/remove installed assets as they are added/removed

* update types

* uninstall assets when installation fails

* refactor installation to add/remove installed assets as they are added/removed

* update types

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

KOTungseth committed:
[DOCS] Fixes to API docs (#71678)

* [DOCS] Fixes to API docs

* Fixes rogue -u

cjcenizal committed:
Fix mappings for Upgrade Assistant reindexOperationSavedObjectType. (#71710)

MadameSheema committed:
removes timeline callout (#71718)

shahzad31 committed:
[Uptime] Visitors breakdowns and enable rum view only via URL (#71428)

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

mbondyra committed:
[Lens] XY chart -long legend overflows chart in editor Feature:Lens (#70702)

tsullivan committed:
[Reporting] Move a few server files for shorter paths (#71591)

nchaulet committed:
[Ingest Manager] Copy change enroll new agent -> Add Agent (#71691)

kevinlog committed:
adjust query to include agents without endpoint as unenrolled (#71715)

tylersmalley committed:
[tests] Temporarily skipped to promote snapshot

Will be re-enabled in https://github.com/elastic/kibana/pull/71727

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

alisonelizabeth committed:
[Index Management] Adopt data stream API changes (#71682)

Commit:
Use updated onPreAuth from Platform (#71552)

* Use updated onPreAuth from Platform

* Add config flag. Increase default value.

* Set max connections flag default to 0 (disabled)

* Don't use limiting logic on checkin route

* Confirm preAuth handler only added when max > 0

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

stephmilovic committed:
[Security solution] [Hosts] Endpoint overview on host details page (#71466)

KOTungseth committed:
[DOCS] Adds API keys to API docs (#71738)

* [DOCS] Adds API keys to API docs

* Fixes link title

* Update docs/api/using-api.asciidoc

Co-authored-by: Brandon Morelli <bmorelli25@gmail.com>

Co-authored-by: Brandon Morelli <bmorelli25@gmail.com>

nreese committed:
[Maps] fix custom icon palettes UI not being displayed (#71482)

* [Maps] fix custom icon palettes UI not being displayed

* cleanup test

* remove uneeded change to vector style defaults

* fix jest tests

* review feedback

* fix jest tests

KOTungseth committed:
[DOCS] Adds ID to logstash pipeline (#71726)

cnasikas committed:
[SIEM] Fix custom date time mapping bug (#70713)

Co-authored-by: Xavier Mouligneau <xavier.mouligneau@elastic.co>
Co-authored-by: Xavier Mouligneau <189600+XavierM@users.noreply.github.com>
Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

spong committed:
[Security] Adds field mapping support to rule creation Part II (#71402)

## Summary

Followup to https://github.com/elastic/kibana/pull/70288, which includes:

- [X] Rule Execution logic for:
  - [X] Severity Override
  - [X] Risk Score Override
  - [X] Rule Name Override
  - [X] Timestamp Override
- [X] Support for toggling display of Building Block Rules:
  - [X] Main Detections Page
  - [X] Rule Details Page
- [X] Integrates `AutocompleteField` for:
  - [X] Severity Override
  - [X] Risk Score Override
  - [X] Rule Name Override
  - [X] Timestamp Override
- [X] Fixes rehydration of `EditAboutStep` in `Edit Rule`
- [X] Fixes `Rule Details` Description rollup


Additional followup cleanup:
- [ ] Adds risk_score` to `risk_score_mapping`
- [ ] Improves field validation
- [ ] Disables override fields for ML Rules
- [ ] Orders `SeverityMapping` by `severity` on create/update
- [ ] Allow unbounded max-signals


### Checklist

Delete any items that are not applicable to this PR.

- [X] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)
- [ ] [Documentation](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#writing-documentation) was added for features that require explanation or tutorials
  - Syncing w/ @benskelker
- [X] [Unit or functional tests](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility) were updated or added to match the most common scenarios
### For maintainers

- [X] This was checked for breaking API changes and was [labeled appropriately](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#release-notes-process)

chrisronline committed:
[Monitoring] Out of the box alerting (#68805)

* First draft, not quite working but a good start

* More working

* Support configuring throttle

* Get the other alerts working too

* More

* Separate into individual files

* Menu support as well as better integration in existing UIs

* Red borders!

* New overview style, and renamed alert

* more visual updates

* Update cpu usage and improve settings configuration in UI

* Convert cluster health and license expiration alert to use legacy data model

* Remove most of the custom UI and use the flyout

* Add the actual alerts

* Remove more code

* Fix formatting

* Fix up some errors

* Remove unnecessary code

* Updates

* add more links here

* Fix up linkage

* Added nodes changed alert

* Most of the version mismatch working

* Add kibana mismatch

* UI tweaks

* Add timestamp

* Support actions in the enable api

* Move this around

* Better support for changing legacy alerts

* Add missing files

* Update alerts

* Enable alerts whenever any page is visited in SM

* Tweaks

* Use more practical default

* Remove the buggy renderer and ensure setup mode can show all alerts

* Updates

* Remove unnecessary code

* Remove some dead code

* Cleanup

* Fix snapshot

* Fixes

* Fixes

* Fix test

* Add alerts to kibana and logstash listing pages

* Fix test

* Add disable/mute options

* Tweaks

* Fix linting

* Fix i18n

* Adding a couple tests

* Fix localization

* Use http

* Ensure we properly handle when an alert is resolved

* Fix tests

* Hide legacy alerts if not the right license

* Design tweaks

* Fix tests

* PR feedback

* Moar tests

* Fix i18n

* Ensure we have a control over the messaging

* Fix translations

* Tweaks

* More localization

* Copy changes

* Type

lukeelmers committed:
[expressions] AST Builder (#64395)

nreese committed:
[Maps] increase DEFAULT_MAX_BUCKETS_LIMIT to 65535 (#70313)

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

spalger committed:
remove unnecessary context reference from trigger job

(cherry picked from commit 817fdf9b439e85c3ddfda126b3efb4e45c36006b)

dhurley14 committed:
[Security Solution] [DETECTIONS] Set rule status to failure only on large gaps (#71549)

* only display gap error when a gap is too large for the gap mitigation code to cover, general code cleanup, adds some tests for separate function

* removes throwing of errors and log error and return null for maxCatchup, ratio, and gapDiffInUnits properties

* forgot to delete commented out code

* remove math.abs since we fixed this bug by switching around logic when calculating gapDiffInUnits in getGapMaxCatchupRatio fn

* updates tests for when a gap error should be written to rule status

* fix typo

spalger committed:
fix triggered job name

Commit:
re-fix navigate path for master add SAML login to login_page (#71337)

bkimmel committed:
[Security_Solution][Bug] Handle non-ecs categories in events (#71714)

* Make resolver related event categories permissive

tylersmalley committed:
[tests] Temporarily skipped to promote snapshot

Will be re-enabled in #71727

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

thomasneirynck committed:
[Maps] Remove .mvt feature flag (#71779)

The layer wizard to add 3rd party .mvt tiles now shows by default.

Commit:
Unskip functional tests for feature controls (#71173)

* Unskip functional tests for feature controls

* Update Maps test

* Update test title

* Fix hidden case-sensitive issue in saved queries

* Fix test separation issues

* Improve saved query retry logic

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

Commit:
[kbn/optimizer] report sizes of assets produced by optimizer (#71319)

* Revert "Report page load asset size (#66224)"

This reverts commit 6f57fa0b2d12e87abab528b60a0da20495b1fb3e.

* [kbn/optimizer] report sizes of assets produced by optimizer

* coalese the fast-glob versions we're using to prevent additional installs

* update kbn/pm dist

* Revert "update kbn/pm dist"

This reverts commit 68e24f0fadd545d649663fd5cbeb98c50ea84dc3.

* Revert "coalese the fast-glob versions we're using to prevent additional installs"

This reverts commit 4201fb60b66bf59dd9e50dab9d0ff66131df8974.

* remove fast-glob, just recursivly call readdirSync()

* update integration tests to use new chunk filename

Co-authored-by: spalger <spalger@users.noreply.github.com>
Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

peteharverson committed:
[ML] Edits labelling of SIEM module and jobs from SIEM to Security (#71696)

## Summary

Edits all references to 'SIEM' in the ML SIEM modules to 'Security'. The following parts of the configurations were edited:
- Module titles
- Module descriptions
- Job descriptions
- `siem` job group changed to `security`

The `siem#/` portion of the custom URLs was also edited to `security/`.

Also removes the 'beta' label from module and job descriptions.

![image](https://user-images.githubusercontent.com/7405507/87452224-dbe4fd00-c5f8-11ea-887b-89c47e3467d2.png)

![image (26)](https://user-images.githubusercontent.com/7405507/87452265-edc6a000-c5f8-11ea-94a8-e101126666fa.png)


Part of #69319

tylersmalley committed:
[tests] Temporarily skipped to promote snapshot

Will be re-enabled in #71727

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

nchaulet committed:
[Ingest Manager] Better display of Fleet requirements (#71686)

dhurley14 committed:
[Security Solution] [Detections] Fixes bug for determining when we hit max signals after filtering with lists (#71768)

update signal counter with filtered results, not with direct search results.

tsullivan committed:
[Reporting] Re-delete a file (#71730)

...that was accidentally recovered due to incorrect manual merge

dplumlee committed:
[Security Solution][Exceptions] - Adds filtering to endpoint index patterns by exceptional fields (#71757)

rylnd committed:
Skip jest tests that timeout waiting for react (#71801)

marshallmain committed:
Add endpoint exception creation API validation (#71791)

rylnd committed:
[Security Solution][Detections] Associate Endpoint Exceptions List to Rule during rule creation/update (#71794)

* Add checkbox to associate rule with global endpoint exception list

This works on creation, now we need edit.

* Fix DomNesting error on ML Card Description

EuiText generates a div, but this is inside of an EuiCard which is a
paragraph. Defines a span with equivalent styles, instead.

* Change default stack of alerts histogram to signal.rule.name

tylersmalley committed:
[test] Skipped monitoring test

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

tylersmalley committed:
[tests] Temporarily skipped Fleet tests

Most fleet tests are colliding with the change to timestamp_field ES
change

https://github.com/elastic/kibana/pull/71727

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

tylersmalley committed:
Revert "re-fix navigate path for master add SAML login to login_page (#71337)"

This reverts commit 1f340969eeb2a5f977e1bad28daab5f2fb96a3a0.

tylersmalley committed:
[test] Skips flaky detection engine tests

https://github.com/elastic/kibana/issues/71814

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

tylersmalley committed:
[test] Skips Ingest Manager test preventing ES promotion

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

sorenlouv committed:
[APM] Increase `xpack.apm.ui.transactionGroupBucketSize` (#71661)

sorenlouv committed:
[APM] Remove watcher integration (#71655)

tylersmalley committed:
[test] Skips flaky Saved Objects Management test

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

FrankHassanabad committed:
[SIEM][Detection Engine][Lists] Adds specific endpoint_list REST API and API for abilities to auto-create the endpoint_list if it gets deleted (#71792)

* Adds specific endpoint_list REST API and API for abilities to autocreate the endpoint_list if it gets deleted

* Added the check against prepackaged list

* Updated to use LIST names

* Removed the namespace where it does not belong

* Updates per code review an extra space that was added

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

mshustov committed:
use fixed isChromeVisible method (#71813)

XavierM committed:
[SECURITY] Timeline bug 7.9 (#71748)

* remove delay of rendering row

* Fix flyout timeline to behave as we wanted

* Fix tabs on timeline page

* disable sensor visibility when you have less than 100 events in timeline

* Fix container to fit content and not take all the place that it wants

* do not update timeline time when switching top nav

* fix timeline url in case

* review I

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

efreeti committed:
Fixed the spacing of child accordion items for policy response dialog. (#71677)

tylersmalley committed:
[tests] Temporarily skipped to promote snapshot

Will be re-enabled in #71727

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

watson committed:
Add @elastic/safer-lodash-set as an alternative to lodash.set (#67452)

lukasolson committed:
[Search] Add telemetry for data plugin search service (#70677)

* [search] Refactor the way search strategies are registered/retrieved on the server

* Fix types and tests and update docs

* Fix failing test

* Fix build of example plugin

* Fix functional test

* Make server strategies sync

* Move strategy name into options

* docs

* Remove FE strategies

* TypeScript of hell
delete search explorer

* Fix search interceptor OSS tests

* typos

* test cleanup

* Update search interceptor tests and abort utils

* [Search] Add telemetry for data plugin search service

* Add tracking of average query time

* Add tests and rename to collectors

* Fix TS

* Fixed interceptor jest tests

* Add to kibana json

* docs

* Properly use observables rather than only during setup

* Update or create

* Swallow version conflict errors

Co-authored-by: Liza K <liza.katz@elastic.co>
Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

rw-access committed:
[Detection Rules] Add 7.9 rules (#71808)

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

peluja1012 committed:
[Security Solution][Exception Modal] Create endpoint exception list if it doesn't already exist (#71807)

* use createEndpointList api

* fix lint

* update list id constant

* add schema test

* add api test

andrew-goldstein committed:
[Security Solution] Full screen timeline, Collapse event (#71786)

## Full screen Timeline & Timeline-based views

- Adds a _Full screen_ mode to Timeline, and all Timeline-based views, including:
  - Detections
  - Detections > Rule details
  - Hosts > Events
  - Hosts > External alerts
  - Network > External alerts
  - Timeline
- Enter full screen from any Resolver
- Adds a `Collapse event` action for quickly collapsing an expanded Timeline event
- Hides the `Add to case action` in timeline-based Resolver views, so those actions are only enabled in Timeline (a `TODO`  from https://github.com/elastic/kibana/pull/70111)

### Full screen detections
![full-screen-detections](https://user-images.githubusercontent.com/4459398/87493332-d348f280-c609-11ea-9399-126d2259daa2.gif)

### Enter full screen from any Resolver
![full-screen-resolver](https://user-images.githubusercontent.com/4459398/87493348-de038780-c609-11ea-86a3-52ab24055e38.gif)

### Full screen Timeline
![full-screen-timeline](https://user-images.githubusercontent.com/4459398/87493394-f4114800-c609-11ea-8d62-4add291d937a.gif)

### Collapse event
![collapse-event](https://user-images.githubusercontent.com/4459398/87493408-fa9fbf80-c609-11ea-88c8-fa87d82d1eb1.gif)

### Sort tooltip
![sort-tooltip](https://user-images.githubusercontent.com/4459398/87493417-012e3700-c60a-11ea-9905-44e3b7cfe60f.gif)

yctercero committed:
[Security Solution][Detection Engine] - Update exceptions logic (#71512)

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>
Co-authored-by: Yara Tercero <yara.tercero@elastic.co>

smith committed:
[APM] Add error rates to Service Map popovers (#69520)

Make the `getErrorRate` function used in the error rate charts additionally take `service.environment` as a filter and have it return the `average` of the values.

Call that function in the API for the service map metrics.

Fixes #68160.

Co-authored-by: cauemarcondes <caue.marcondes@elastic.co>

darnautov committed:
[ML] Fix UI Actions context menu positioning for the Anomaly Swim Lane (#71839)

* [ML] fix swim lane embeddable rerenders

* [ML] fix TS

ashikmeerankutty committed:
Convert vis_type_vega to Typescript (#68915)

michaelolo24 committed:
add policy details and update SO limit requests (#71789)

jbudz committed:
[logging] Format new platform json logging to ECS (#71138)

* [logging] Format new platform json logging to ECS

* update integration tests

* merge instead of assign

* add @timestamp override test

* add partial merge test against log object

* add object level override test

* fix type error

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

tylersmalley committed:
Restores task for downloading Chromium builds (#71749)

This was removed in https://github.com/elastic/kibana/pull/69165 without
realizing it was used by the packer cache. I renamed it to be more
inline with what it actually does.

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

efreeti committed:
Fixed the beta badge layout. (#71835)

Commit:
add short sleep before clicking Remove on sample data (#71104)

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

Dosant committed:
[uiActions] Support emitting nested triggers and actions (#70602)

* Introduce automatically executed actions
* Introduce batching of emitted triggers to be execute on the macro task

loixlab committed:
[Form lib] Memoize form hook object and fix hook array deps (#71237)

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>
Co-authored-by: Patryk Kopycinski <contact@patrykkopycinski.com>

dej611 commented:
@elasticmachine merge upstream

tylersmalley committed:
Removes timestamp_field from data_stream (#71727)

https://github.com/elastic/kibana/issues/71670

Caused by https://github.com/elastic/elasticsearch/pull/59317

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

dej611 commented:
@elasticmachine merge upstream

elasticmachine commented:
expected head sha didn’t match current head ref.

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/61980/)
* Commit: be39e89397b98e79b0dccd191755661fad0b2f83

### Build metrics
#### miscellaneous assets size
| id | [value](https://github.com/elastic/kibana/commit/be39e89397b98e79b0dccd191755661fad0b2f83) | diff | [baseline](https://github.com/elastic/kibana/commit/6068285c378d3b83cc97b8b20ca47abb273e2ba3) |
| --- | --- | --- | --- |
| upgradeAssistant | 22.5KB | -25.0B | 22.6KB |
| visTypeVislib | 505.4KB | -6.0B | 505.4KB |
| total | - | -31.0B | - |

#### page load bundle size
| id | [value](https://github.com/elastic/kibana/commit/be39e89397b98e79b0dccd191755661fad0b2f83) | diff | [baseline](https://github.com/elastic/kibana/commit/6068285c378d3b83cc97b8b20ca47abb273e2ba3) |
| --- | --- | --- | --- |
| visTypeVislib | 1.3MB | +125.0B | 1.3MB |

### History
* :green_heart: [Build #61277](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/61277/) succeeded 17d7e6e69aaadf46c33cefcb8c1c8f915c0e6f9b

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/61277/","number":"61277","commit":"17d7e6e69aaadf46c33cefcb8c1c8f915c0e6f9b"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/61980/","number":"61980","commit":"be39e89397b98e79b0dccd191755661fad0b2f83"}]}
PIPELINE-->
